packages/dx-react-grid-demos/src/demo-sources/core-basic/bootstrap3/plugin-host-component.jsx

```diff
@@ -1,30 +1,22 @@
-import * as React from 'react';
+import React, { useState } from 'react';
```

Comment:
Why we change import style?

Reply:
We need to call useState. We can do it different ways:
1) "React.useState"
2) import useState with React in same line
3) import useState with React in different lines
I choosed case 2, but then, we can't use "* as React" construction